### PR TITLE
Custom exceptions2

### PR DESCRIPTION
**Pull request by Gordo851**
Custom exception added and test to check if it works. Also new CSV file to allow testing of exception.

### CODE REVIEW

File: CW1/src/customExceptions/InvalidProductPriceException.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package customExceptions;
+
+public class InvalidProductPriceException extends Exception {
+	public InvalidProductPriceException(String message) {
+		super(message);
+	}
+
+	}
```
Comment (MaxAtkinson) [inline CW1/src/customExceptions/InvalidProductPriceException.java:8]: This bracket should be at the root level

File: CW1/src/testing/MemoribiliaTest.java [hunk 2]
```diff
@@ -19,7 +20,7 @@
 
 	
 	@Before
-	public void setup(){
+	public void setup() throws InvalidProductPriceException{
 	m = new Memoribilia(name, desc, price,id);
 	}
```
Comment (MaxAtkinson) [inline CW1/src/testing/MemoribiliaTest.java:26]: And here

File: CW1/src/testing/FoodClassTests.java [hunk 3]
```diff
@@ -19,7 +20,7 @@
 
 	
 	@Before
-	public void setup(){
+	public void setup() throws InvalidProductPriceException{
 	f = new Food(name, desc, price,id);
 	}
```
Comment (MaxAtkinson) [inline CW1/src/testing/FoodClassTests.java:26]: Bracket

**Review by aln3 — APPROVED** [review]
Reviewed and approved.

**Pull request merged**